ctapipe/calib/camera/calibrator.py

```diff
@@ -35,13 +35,14 @@ def integration_correction(
     reference_pulse_shape : ndarray
         Numpy array containing the pulse shape for each gain channel
     reference_pulse_step : float
-        The step in time for each sample of the reference pulse shape
-    sampled_step : int
-        The step in time for each sample of the sampled waveforms
+        The step in time for each sample of the reference pulse shape in ns
+    sample_width : float
```

Comment:
one more pedantic comment, if *not* using astropy quantities here, you should name it `sample_width_ns`.  Otherwise, looks good.